randomMenuLunch.rb

```diff
@@ -0,0 +1,29 @@
+# Random Menu Generator (RMG Lunch)
+
+descriptor1 = ["fresh", "bite-size", "aromatic", "aged", "famous", "classic", "delicious", "delightful", "fantastic", "traditional"]
+descriptor2 = ["baked", "boiled", "caramelized", "creamy", "crunchy", "rosted", "sliced", "smoked", "grilled", "crisp"]
+ingredient = ["eggs", "cheese", "chicken", "tomatoes", "steak", "salmon", "beef", "garlic", "mushrooms", "sausage"]
+
+(1..ingredient.length).each do |i|
```

Comment:
Neat, I didn't know you could call each on a range. Calling `times` on the length of ingredients would be a little simpler to read, but both totally work.